src/book.js

```diff
@@ -80,6 +89,12 @@ export class Book {
     this.page_positioning = configuration.pagePositioning;
     this.flyleafs = configuration.flyleafs;
     this.cropmarks = configuration.cropMarks;
+    this.frenchStichLink = {
```

Comment:
Nit pick "Stich" is mispelled. I would prefer this be name stitchMarks or something similar. French Link is one specific stitch and not the only one.